.theia/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
You should add `.theia` to the `.gitignore`. Is there a reason to track this file in the main repository?